test/client/index.js

```diff
@@ -463,6 +463,12 @@ module.exports = config => {
       .then(this._clear.bind(this))
   }
 
+  Client.prototype.destroyAccountWithSessionToken = function () {
+    return this.api.accountDestroyWithSessionToken(this.email, this.authPW, this.sessionToken)
```

Comment:
No biggie, but could you also make this conditional behaviour of `destroyAccount`? So it checks `this.sessionToken` internally then decides with to call `this.api.accountDestroy` or `this.api.accountDestroyWithSessionToken`?